package.json

```diff
@@ -73,7 +74,7 @@
     "build": "gatsby build",
     "build-ci": "gatsby build --prefix-paths",
     "start": "gatsby develop",
-    "format": "prettier --write \"**/*.{ts,tsx,js,scss}\" && npm run lint:fix",
+    "format": "prettier --write \"**/*.{ts,tsx,js,scss,yml}\" && npm run lint:fix",
```

Comment:
This change makes sense, although it could just be dropped to `.` instead of the big file list. The eslint plugin and changes seem unessessary to add on top

Reply:
Hello @nschonni, I tried changing this to just `.`, it results in hundreds of .md files being formatted. I'm guessing that's why this list was written explicitly.

Reply:
Right, I forgot that the markdown probably would have issues with the wider glob pattern